apps/talk_signaling.sh

```diff
@@ -43,8 +45,6 @@ fi
 
 # Install
 . /etc/lsb-release
-is_process_running dpkg
```

Comment:
why did you remove this? not neccessary anymore? I've copied this over from another script.

Reply:
I moved it to the top simce the first installation happens there.